src/guide/forms.md

```diff
@@ -1,81 +1,81 @@
-# Form Input Bindings
+# Interligações em Formulários
 
-## Basic Usage
+## Uso Básico
 
-You can use the `v-model` directive to create two-way data bindings on form input, textarea, and select elements. It automatically picks the correct way to update the element based on the input type. Although a bit magical, `v-model` is essentially syntax sugar for updating data on user input events, plus special care for some edge cases.
+Você pode usar a diretiva `v-model` para criar interligações de dados de mão dupla (two-way data binding) em elementos input, textarea e select de formulários. Ele automaticamente escolhe a maneira correta de atualizar o elemento baseado no tipo do _input_. Embora um pouco mágico, `v-model` é essencialmente uma forma simplificada(syntax sugar) para atualizar dados nos eventos de entrada do usuário, além de cuidado especial com casos extremos.
```

Comment:
É bacana deixar o original, em inglês, após a tradução de termos técnicos. Poderia apenas deixá-los, além de entre parênteses, em itálico também, por gentileza? Exemplo:

interligações de dados de mão dupla _(two-way data binding)_
```
interligações de dados de mão dupla _(two-way data binding)_
```

é essencialmente uma forma simplificada(syntax sugar)
```
é essencialmente uma forma simplificada _(syntax sugar)_
```

Particularmente, achei interessante a adaptação de _syntax sugar_ como "forma simplificada".